### PR TITLE
Added recipe for wsgiref

### PR DESCRIPTION
**Pull request by pmlandwehr**

### CODE REVIEW

File: recipes/wsgiref/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  run:
+    - python
+
+test:
+  imports:
+    - wsgiref
+
+about:
+  home: http://cheeseshop.python.org/pypi/wsgiref
+  license: PSF or ZPL
```
Comment (jakirkham) [inline recipes/wsgiref/meta.yaml]: Versions?

Comment (pmlandwehr) [inline recipes/wsgiref/meta.yaml]: Nope - all I've got is [what's on pypi](https://pypi.python.org/pypi/wsgiref) - if there's a better source we should definitely use it.

Comment (jakirkham) [inline recipes/wsgiref/meta.yaml]: Legit. Don't see anything else either (not even a license file).

**Comment by jakirkham** [comment]
Thanks @pmlandwehr.

**Pull request merged**